spec/jobs/user_email_spec.rb

```diff
@@ -563,7 +563,7 @@
           Jobs::UserEmail.new.message_for_email(
             user,
             post,
-            :user_mentioned,
+            ActiveSupport::StringInquirer.new("user_mentioned"),
```

Comment:
That feels weird 🤔

Reply:
Yes indeed, that’s because we’re testing an internal method that should be private instead of running the whole job as we do in the rest of the spec. I’ll have a look on how to improve that.